Assets/Chunk.cs

```diff
@@ -53,13 +55,24 @@ private void Start()
 					low: 0,
 					high: 16
 				);
+				/*
+				 * TODO: We can optimize this later by first checking all 4 chunk pages.
+				 * If all are in the same region, all blocks in that chunk are in the same region.
+				 */
+				float deltaVoronoi = (float)ChunkGenerator.NoiseGenerator
+					.Eval(noiseX / 4, noiseZ / 4) * 32;
```

Comment:
The divider 4 for scaling and the multiplier 32 for the value are magic numbers that produce a good result